ui/desktop/app/components/hidden-secret/index.hbs

```diff
@@ -0,0 +1,11 @@
+<div class='hidden-secret'>
+  {{! TODO: The span wrap within a div because later we will add the toggle secret icon/button}}
+  <div>
+    <span>{{t 'resources.target.host-set.secret_label'}}</span>
+  </div>
```

Comment:
This content can be wrapped with <Copyable>.

Reply:
https://gph.is/g/Zx1WqN7